src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerContext.Known.cs

```diff
@@ -76,6 +76,38 @@ public JsonTypeInfo<DateTimeOffset> DateTimeOffset
             }
         }
 
+        private JsonTypeInfo<DateTime>? _dateTime;
+        private static JsonTypeInfo<DateTime>? s_dateTime;
+        /// <summary>
+        /// todo
+        /// </summary>
+        public JsonTypeInfo<DateTime> DateTime
+        {
+            get
+            {
+                if (_dateTime == null)
+                {
+                    // todo: support obtaining existing converter
+                    //if (_options.HasCustomConverters)
+                    //{
+                    //JsonConverter<DateTime> converter = (JsonConverter<DateTime>)_options.GetConverter(typeof(DateTime));
+                    //_dateTime = new JsonValueInfo<DateTime>(converter, _options);
+                    //}
```

Comment:
You can remove this line and create a new issue to track this.

Reply:
https://github.com/dotnet/runtimelab/issues/68 created.

Reply:
nit: the comment is still here

Reply:
Previously I deleted this for Collections and not for values. Deleted now.